frontend-react/src/components/FileHandlers/FileHandler.test.tsx

```diff
@@ -1,4 +1,4 @@
-import { screen, fireEvent, waitFor } from "@testing-library/react";
+import { screen, fireEvent } from "@testing-library/react";
```

Comment:
This entire file has been rewritten (see [`mockUseFileHandler`](https://github.com/CDCgov/prime-reportstream/pull/8201/files#diff-350227b22d21936220afceaa87bbde022566cf5fe7ab301e53da9e0dcbc715b5R126), [`mockUseSenderSchemaOptions`](https://github.com/CDCgov/prime-reportstream/pull/8201/files#diff-350227b22d21936220afceaa87bbde022566cf5fe7ab301e53da9e0dcbc715b5R138), et cetera) as convenience functions to reduce redundancy in rendering the component.  Also, there were some tests that were either 1) not particularly useful or 2) not within the purview of this component.  For example, [checking the alert class](https://github.com/CDCgov/prime-reportstream/pull/8201/files#diff-350227b22d21936220afceaa87bbde022566cf5fe7ab301e53da9e0dcbc715b5L273) is not really useful -- we should have a data attribute if we really need to know the "type" of alert so as to not couple CSS with JavaScript logic -- and that's the responsibility of the StaticAlert component anyway.